soundbay/conf/optim/jasco_resnet_50_finetune.yaml

```diff
@@ -0,0 +1,12 @@
+# @package _global_
+optim:
+  epochs: 100
+  optimizer:
+    _target_: torch.optim.SGD
+    lr: 0.001
+    momentum: 0.9
+    weight_decay: 0.0001
+  scheduler:
+    _target_: torch.optim.lr_scheduler.ExponentialLR
+    gamma: 0.95
+  freeze_layers_for_finetune:  # Default is False. If True, freeze classifier + last layer/block
```

Comment:
should it be
`freeze_layers_for_finetune: False`?
does it run without a value?

Reply:
I also wonder if it's more a `optim` arg or `experiment`. I don't have a strong opinion, think either are fine